resources/definitions/adimlab_gantry.def.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "ADIMLab Gantry",
+    "version": 2,
+    "inherits": "adimlab_base",
```

Comment:
If there aren't actually multiple machines using this, please merge this into a single file